cli/src/main/java/com/devonfw/tools/ide/tool/mvn/Mvn.java

```diff
@@ -30,4 +34,32 @@ public boolean install(boolean silent) {
     return super.install(silent);
   }
 
+  @Override
+  public void installPlugin(PluginDescriptor plugin) {
+
+    Path mavenPlugin = this.context.getSoftwarePath().resolve(this.tool).resolve("lib/ext/" + plugin.getName() + ".jar");
+    System.out.println(getPluginUrl(plugin));
+    this.context.getFileAccess().download(getPluginUrl(plugin), mavenPlugin);
```

Comment:
We could also kick out the `version` attribute from the plugin descriptor.
If you support it, you need to handle the case that the pluginUrl is not specified and compute it from pluginId + version including resolving latest version if version is also undefined.
We could simplify by leaving this out for a future enhancement PR.